pkg/controllers/provisioning/scheduling/nodeclaim.go

```diff
@@ -283,11 +282,6 @@ func fits(instanceType *cloudprovider.InstanceType, requests v1.ResourceList) bo
 }
 
 func hasOffering(instanceType *cloudprovider.InstanceType, requirements scheduling.Requirements) bool {
```

Comment:
nit: im thinking since it's only the two lines here, it may be fine to remove this

Reply:
+1, we can probably inline this function now 🎉